Makefile

```diff
@@ -197,7 +198,7 @@ e2e-test: vet fmt build-tar
 	-boskos-project-type=$(BOSKOS_PROJECT_TYPE) -job-name=$(JOB_NAME) \
 	-artifacts-dir=$(ARTIFACTS)
 
-build-binaries: ./bin/node-problem-detector ./bin/log-counter ./bin/health-checker
+build-binaries: ./bin/node-problem-detector ./bin/log-counter ./bin/health-checker $(WINDOWS_BINARIES) $(LINUX_BINARIES)
```

Comment:
`./bin/node-problem-detector ./bin/log-counter ./bin/health-checker` can be removed?

Reply:
Eventually. There needs to be some updates through the upstream CI scripts to manage this.